spock-core/core.gradle

```diff
@@ -49,20 +47,21 @@ jar {
       'Implementation-Vendor': 'spockframework.org',
       'Automatic-Module-Name': 'org.spockframework.core'
     )
-    instruction 'Export-Package', 'org.spockframework.*', 'spock.*'
-    instruction 'Embed-Dependency', 'groovy;inline=false', 'junit;inline=false'
-    instruction 'Import-Package',
-        'groovy.*;version="[2.4,4)"',
-        'org.codehaus.groovy.*;version="[2.4,4)"',
-        'org.junit.*;version="[4,6)"',
-        'org.hamcrest.*;version="[1,2)";resolution:=optional',
-        'org.objenesis.*;version="[2,3)";resolution:=optional',
-        'org.apache.tools.ant.*;version="[1.7,2)";resolution:=optional',
-        'net.bytebuddy.*;version="[1.9,2)";resolution:=optional',
-        'net.sf.cglib.*;version="[3.2,4)";resolution:=optional',
-        'org.objectweb.asm.*;version="[5.2,6)";resolution:=optional',
-        'org.w3c.dom.*'
   }
+  bnd(
+    'Export-Package': ['org.spockframework.*', 'spock.*'].join(','),
+    'Embed-Dependency': ['groovy;inline=false', 'junit;inline=false'].join(','),
```

Comment:
I wonder, if the second holds also for junit5?

Reply:
I've actually no idea what that means, I just converted it.

Reply:
Sure, this PR is mainly about migration, not extra improving. We can create a separate issue to handle that later of (unless @rotty3000 will help us ;-) ).

Reply:
Gaving it a short Google search it was probably always useless.
According to search results, the `Embed-Dependency` instruction is only interpreted by the `Apache Felix Maven Bundle` maven plugin and would cause it to build the bundle jar as fat jar including the dependencies declared in the POM.
As far as I know neither the Gradle built-in plugin, nor the community plugin interprets this instruction but only bring the JARs manifest in shape, so yes, you should probably just remove it.

Reply:
Hello everyone! Without blocking your current progress, if you give me a little time to look into this I would love to help with spock's OSGi metadata.